### PR TITLE
feat: add schema spy

### PR DESCRIPTION
**Pull request by RMCampos**
# Description
Closes no issue;

#### New
- Docker service , config file, and readme for Schema Spy;

#### Changed
- gitignore file

#### Removed
- None.

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-10-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1210-backend.apps.silver.devops.gov.bc.ca/swagger-ui/index.html)
- [Oracle-API](https://nr-spar-1210-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -121,3 +121,15 @@ services:
       interval: 5s
       timeout: 5s
       retries: 5
+
+  schemaspy:
+    container_name: schemaspy
+    profiles: ["schemaspy"]
+    image: schemaspy/schemaspy:6.2.4
+    user: "${UID}:${GID}"
```
Comment (RMCampos) [inline docker-compose.yml:129]: We need root, sorry. Here's the reported issue, still open: https://github.com/schemaspy/schemaspy/issues/943

File: .gitignore [hunk 2]
```diff
@@ -203,3 +203,6 @@ dist
 # Data
 sync/data/
 **/__pycache__
+
+# Schema Spy output
+schemaspy/output
```
Comment (craigyu) [inline .gitignore:208]: eMpTY LiNE 🤪

**Review by craigyu — APPROVED** [review]
Good stuff! added one nitpicking comment

File: schemaspy/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Schema Spy provides a simple way of generating a great UI, web page, 
+containing all database tables, triggers, views, and more.
+
+It provides a easy way of search for tables, columns, and pretty much
+any information on the database schema definition.
+
+## Running locally
+
+Currently there's no deployed page available. Howerver, with Docker +
```
Comment (mgaseta) [inline schemaspy/README.md:13]: Typo in `However`.

File: schemaspy/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+docker compose --profile schemaspy up -d
+```
+
+With that you should be able to get:
+
+- Postgres Database up and running on port 5432
+- Backend service up and running, including all migrations
+- Schema Spy updated files inside `schemaspy` directory.
+
+Now open up the file `schemaspy/index.html` on yoru browser and enjoy :)
```
Comment (mgaseta) [inline schemaspy/README.md:29]: Typo on `your`.

File: schemaspy/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Config file
+
+The config file can be found at the `schemaspy` directory, named `postgres.propertes`.
+
+## References
+
+- **SchemaSpy on GitHub**: https://github.com/schemaspy/schemaspy
+- **SchemaSpy on Docker Bub**: https://hub.docker.com/r/schemaspy/schemaspy
```
Comment (mgaseta) [inline schemaspy/README.md:38]: Typo on `Hub`

**Review by mgaseta — APPROVED** [review]
Nice job! :100: :1st_place_medal: 
Looking forward to see schemaspy working!
Added only a few non-blocking markups.

File: .gitignore (hunk 2 above)
Comment (RMCampos) [reply .gitignore]: Done https://github.com/bcgov/nr-spar/pull/1210/commits/9bfb54789878a6054f488e2228371dca5c8f63bd

File: schemaspy/README.md (hunk 3 above)
Comment (RMCampos) [reply schemaspy/README.md]: Done https://github.com/bcgov/nr-spar/pull/1210/commits/d59df160079af87b2469118ac0141922e0a36818

File: schemaspy/README.md (hunk 4 above)
Comment (RMCampos) [reply schemaspy/README.md]: Done https://github.com/bcgov/nr-spar/pull/1210/commits/d59df160079af87b2469118ac0141922e0a36818

File: schemaspy/README.md (hunk 5 above)
Comment (RMCampos) [reply schemaspy/README.md]: Done https://github.com/bcgov/nr-spar/pull/1210/commits/d59df160079af87b2469118ac0141922e0a36818

**Pull request merged**